go.mod

```diff
@@ -18,15 +18,14 @@ require (
 	github.com/aws/aws-sdk-go-v2/credentials v1.17.18
 	github.com/aws/aws-sdk-go-v2/service/route53 v1.40.10
 	github.com/cespare/cp v1.1.1
-	github.com/cloudflare/cloudflare-go v0.97.0
-	github.com/cockroachdb/pebble v1.1.2
+	github.com/cloudflare/cloudflare-go v0.114.0
```

Comment:
Shall we take this opportunity to bump the following deps?
- github.com/0xPolygon/heimdall-v2 to `v0.3.1`
- github.com/cosmos/cosmos-sdk v0.50.13 to `v0.50.14` (it gets replaced but it's just to quit some sec warnings)
- github.com/0xPolygon/bor to latest stable version